#Header header

string object_id
float32 distance 	# distance between the object center of gravity and the detected pointing coordinates


